src/common/FrameManager.ts

```diff
@@ -398,7 +398,7 @@ export class FrameManager extends EventEmitter {
             frameId: frame._id,
             worldName: name,
             grantUniveralAccess: true,
-          })
+          }).catch(console.error)
```

Comment:
Earlier this was 

```suggestion
          }).catch(debugError)
```

imported with `import {debugError} from './helper';`. Should it be the debugError instead?

(For some reason it got removed in https://github.com/puppeteer/puppeteer/commit/4d9dc8c0e613f22d4cdf237e8bd0b0da3c588edb)

Reply:
👍 

Looks good now